src/pyfaf/actions/assign_release_to_builds.py

```diff
@@ -57,29 +57,29 @@ def run(self, cmdline, db):
         arch = get_arch_by_name(db, cmdline.ARCH)
         if not arch:
             self.log_error("Selected architecture '{0}' is not supported."
-                            .format(cmdline.ARCH))
+                           .format(cmdline.ARCH))
             return 1
         else:
             self.log_info("Selected architecture: '{0}'".format(cmdline.ARCH))
 
         # when release-builds argument specified
         if cmdline.released_builds:
             self.log_info("Assigning released builds for '{0} {1}'"
-                    .format(cmdline.OPSYS, cmdline.RELEASE))
+                          .format(cmdline.OPSYS, cmdline.RELEASE))
             opsys = self._edit_opsys(cmdline.OPSYS)
             if not opsys in systems.keys():
                 self.log_error("There are no known released builds for '{0}'"
-                                    .format(cmdline.OPSYS))
+                               .format(cmdline.OPSYS))
                 return 1
 
             for build in systems[opsys].get_released_builds(cmdline.RELEASE):
                 found_build = (db.session.query(Build)
-                         .filter(Build.base_package_name ==
-                                 build["name"])
-                         .filter(Build.version == build["version"])
-                         .filter(Build.release == build["release"])
-                         .filter(Build.epoch == build["epoch"])
-                         .first())
+                               .filter(Build.base_package_name ==
+                                       build["name"])
```

Comment:
this break is not really necessary. It is only 81 characters without break - when you rebase, the warning goes away:)